src/components/MovieList/MovieList.jsx

```diff
@@ -1,5 +1,8 @@
+import { MovieCard } from '../MovieCard';
 import './MovieList.scss';
 
-export const MovieList = () => (
-  <>Put the list here</>
+export const MovieList = ({ movies }) => (
+  <div className="movies">
+    {movies.map(movie => <MovieCard movie={movie} key={movie.imdbId} />)}
```

Comment:
it will be better to do it in more readability way
```suggestion
    {movies.map(movie => (
      <MovieCard
        key={movie.imdbId}
        movie={movie}
      />)
    )}
```